game-core/src/test/java/games/strategy/util/IntegerMapTest.java

```diff
@@ -97,17 +89,114 @@ public void testAddMap() {
   }
 
   @Test
-  public void testGreaterThan() {
+  public void testGreaterThanOrEqualTo() {
     final IntegerMap<Object> map1 = new IntegerMap<>();
     map1.add(v1, 5);
     map1.add(v2, 3);
     final IntegerMap<Object> map2 = new IntegerMap<>();
     map2.add(v1, 5);
     map2.add(v2, 3);
     map2.add(v3, 1);
-    assertTrue(!map1.greaterThanOrEqualTo(map2));
+    assertFalse(map1.greaterThanOrEqualTo(map2));
     assertTrue(map2.greaterThanOrEqualTo(map2));
     map1.add(v3, 3);
     assertTrue(map1.greaterThanOrEqualTo(map2));
   }
+
+  @Test
+  public void testGetInt() {
+    final IntegerMap<Object> map = new IntegerMap<>();
+    map.add(v1, 0);
```

Comment:
I know it pre-existed this PR, but `v1`, `v2`, and `v3` would be better named `k1`, `k2`, and `k3`.